MSAL/src/native_auth/public/MSALNativeAuthPublicClientApplication.swift

```diff
@@ -31,6 +31,9 @@ public final class MSALNativeAuthPublicClientApplication: MSALPublicClientApplic
     let inputValidator: MSALNativeAuthInputValidating
     private let internalChallengeTypes: [MSALNativeAuthInternalChallengeType]
 
+    private lazy var cacheAccessor =
```

Comment:
Nit: This can be `internal` as it is reused in the `MSALNativeAuthPublicClientApplication+internal.swift` file. I think it's fine to use it instead of passing it as a parameter, in the end it's just an extension of the same class (similar to what you did in the class `MSALNativeAuthUserAccountResult`).

Reply:
Thanks @diegojerezba . I addressed this comment in the new PR (https://github.com/AzureAD/microsoft-authentication-library-for-objc/pull/1958) and did as you suggested. Great idea.